### ISSUE TITLE
Elden Ring crashing on startup with new patch

### ISSUE THREAD

**Issue by Thekidster1** [soulsmods/ModEngine2#119]
Since the patch today my Elden Ring crashes on startup when using the launch.bat, I take it the patch has screwed with the mod loader?

**Comment by Dealman** [comment]
Are you using Seamless Co-op? It is not yet updated for Elden Ring v1.09;
![image](https://user-images.githubusercontent.com/7038067/227373152-1c672c62-d630-42c4-9691-1ca0b0ee5b41.png)

I'd suggest following the process on their nexus page. :)

**Comment by CrypticVillain** [comment]
Definitely a mod loader issue I have the same problem despite removing all my mods from mod engine. Elden ring works fine when launched from the exe, but if launched from the .bat it will crash while loading the game save.

**Comment by borjpc** [comment]
Hello people, saying issue here. I tried out diferent mods, textures, models, effects... nothing works, it crashes in the loading page at the very begining. 

My game is original, no cracked version. Specially having issues with adding the folder named **'parts'**

Mod needs to update, or mod engine 2? i'm confused. HELP lol

**Comment by borjpc** [comment]
> > Hello people, saying issue here. I tried out diferent mods, textures, models, effects... nothing works, it crashes in the loading page at the very begining.
> > My game is original, no cracked version. Specially having issues with adding the folder named **'parts'**
> > Mod needs to update, or mod engine 2? i'm confused. HELP lol
> 
> Have you tried to launch an online version? You may need to disable ray tracing in the main menu before starting the game

Correct, after a whole day trying stuff, this is the solution for everyone, enter the game, without the .bat. as normally you would do with your exe without mods, disable the ray tracing, then execute the .bat and the mods will work and no crashing problems.

**Comment by Dealman** [comment]
Starting the game once without mods I thought was standard procedure? I'll try it later this evening to verify whether disabling ray tracing is actually necessary.

**Comment by CirnoAlter** [comment]
I've launched once with no mods, going online, and even going back in it crashes. Though, it was on preview 3. I downloaded preview 4 and that one just doesn't launch at all.

**Comment by borjpc** [comment]
> I've launched once with no mods, going online, and even going back in it crashes. Though, it was on preview 3. I downloaded preview 4 and that one just doesn't launch at all.

Enter the game, with the normal exe, as if you were entering the game with no mods from steam, then, remove ray tracing. Close the game. Go to your Mod Engine 2 folder, and execute the .bat file to open the game. This will solve the issue.

**Comment by Fox-parade** [comment]
> > I've launched once with no mods, going online, and even going back in it crashes. Though, it was on preview 3. I downloaded preview 4 and that one just doesn't launch at all.
> 
> Enter the game, with the normal exe, as if you were entering the game with no mods from steam, then, remove ray tracing. Close the game. Go to your Mod Engine 2 folder, and execute the .bat file to open the game. This will solve the issue.

im having the issuse where my mods dont load when launching from the .bat, launched the game normaly and works. using the seamless coop .exe works but using the modengine .bat does not load the coop or randomizer mods. was working fine before the game update.

**Comment by Dealman** [comment]
> im having the issuse where my mods dont load when launching from the .bat, launched the game normaly and works. using the seamless coop .exe works but using the modengine .bat does not load the coop or randomizer mods. was working fine before the game update.

Did you verify your config is correctly setup? Randomizer changes quite a long of stuff, maybe double-check if there's a new version for it?

**Comment by Thekidster1** [comment]
Initial problem was actually just Seamless, Just update and you should be good to go (After the usual update procedures)

**Comment by Dealman** [comment]
> I can safely tell you there is no problem with ModEngine2. It all comes down to user error and/or Fromsoft mucking things up in this very unstable and crashy update.

Can confirm, simply making sure you start the game without mods first and then Seamless Co-op if you have it _(via its own executable)_ and _then_ ModEngine 2 - should work.

Ray tracing does not have to be disabled.

**Comment by Gulciane** [comment]
raytracing does not work with mod engine 2, just disable it in the options, maybe a mod engine 2 update to fix it
